surveys/2022-annual-survey/questions.md

```diff
@@ -401,22 +468,25 @@ Type: select all that apply (optional)
 - Blockchain
 - Cloud computing applications
 - Cloud computing infrastructure or utilities
+- CLI
```

Comment:
I would spell this out. I've talked with folks experienced in the industry who don't know the term CLI as "command line interface".